### ISSUE TITLE
Web UI: Data Column selection should be persistent

### ISSUE THREAD

**Issue by BobEvans** [google/paco#1229]
From a senior researcher:

"When I select certain columns to show in Data, I expect that this will be persistent, and I don't have to re-select them every time I go back to the view."

We should find a way, other than the share link to make this persistent.

**Comment by ianspiro** [comment]
To me that suggests cookies. I don't think it's worth adding any kind of server piece to store this per account, or at least not as a first resort. Also, we probably still want the hyperlinks to address Mike's use-case.

**Comment by BobEvans** [comment]
Did we solve this already?

**Comment by ianspiro** [comment]
No. This would require cookies or storing data with a user account. I'm
hesistant to add cookies at this point. Maybe we can validate that this
would be useful enough to researchers.
